devops/README.md

```diff
@@ -0,0 +1,71 @@
+# OSConfig DevOps
+This folder contains various folders in use by OSConfig's Engineering Systems (Build/Packaging/Test Infrastructure). This readme serves to describe the various components as they have been created with reusability in mind.
+
+ * debian
+ * docker
+ * pipeline
+ * scripts
+ * terraform
```

Comment:
Uppercase for names first letters?

Reply:
i wanted to match the folder names, which are case-sensitive